v2/openapi/fb-unified-openapi.yaml

```diff
@@ -2905,13 +2933,18 @@ components:
               type: string
             amount:
               $ref: '#/components/schemas/PositiveAmount'
-    CrossAccountTransfer:
+    PeerAccountTransfer:
       type: object
       allOf:
-        - $ref: '#/components/schemas/CrossAccountTransferDestination'
+        - $ref: '#/components/schemas/PeerAccountTransferDestination'
```

Comment:
Add optional reference ID as in fiat transfers.

Reply:
To the InternalTransfer as well?

Reply:
Done